testsuite/test-generate-zeppelin.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+source test-generate-common.sh
+source test-common.sh
+source test-config.sh
+
+__GenerateZeppelinStandardTests_BasicTests() {
+    local sparkversion=$1
+    local javaversion=$2
+    local zeppelinversion=$3
+    local zeppelinhome=$4
+
+    cp ../submission-scripts/script-${submissiontype}/magpie.${submissiontype}-spark-with-zeppelin magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
+    sed -i -e 's|export MAGPIE_JOB_TYPE="\(.*\)"|export MAGPIE_JOB_TYPE="script"|' magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
+    sed -i -e 's|export SPARK_VERSION="\(.*\)"|export SPARK_VERSION="'"${sparkversion}"'"|' magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
+    sed -i -e 's|export SPARK_MODE="\(.*\)"|export SPARK_MODE="script"|' magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
+    sed -i -e 's|# export SPARK_SCRIPT_PATH="\(.*\)"|export SPARK_SCRIPT_PATH="testscripts/test-zeppelin.py"|' magpie.${submissiontype}-spark-with-zeppelin-${zeppelinversion}-${sparkversion}
+
```

Comment:
shouldn't this be "MAGPIE_SCRIPT_PATH" since we're using MAGPIE_JOB_TYPE=script?